src/store/Store.ts

```diff
@@ -30,13 +30,21 @@ export interface ServiceGraphDataState {
   graphData: any;
 }
 
+export interface ServiceGraphState {
+  dataState: ServiceGraphDataState;
+  filterState: ServiceGraphFilterState;
+  selectedObject: {
```

Comment:
`Object` is another one of those general/overloaded terms. Almost everything is an object. The name should convey what kind of _selectedObject_ it is.